app/models/school.rb

```diff
@@ -584,6 +584,10 @@ def subscription_frequency
     end
   end
 
+  def recent_usage
```

Comment:
Just noting that we have a service that does mostly this already: `ManagementTableService`. Wondering if it'd be best to align, so there's one way to load the data?

Reply:
yep no probs.  I've updated to use `ManagementTableService`